pinot-common/src/main/java/com/linkedin/pinot/common/utils/CommonConstants.java

```diff
@@ -95,6 +95,8 @@
         public static final String REALTIME_SEGMENT_FLUSH_TIME = "realtime.segment.flush.threshold.time";
         // Num records threshold in the realtime segment
         public static final String REALTIME_SEGMENT_FLUSH_SIZE = "realtime.segment.flush.threshold.size";
+        // Flush threshold size across all segments
```

Comment:
"flush threshold size across all segments of one table in one server". I suggest you add some note on how to set this value (so we don't have to read the code to figure out how it is used). e.g. something like:
"Data is stored in heap memory while rows are being consumed from kafka in realtime tables. If your servers are handling multiple tables you need to split the heap memory among the tables. Assuming you have arrived at a value of X GB for the table, and the row size is Y, you can set this value to X/Y for that table."

A note about what should be done if both HLC and LLC are in place would also be helpful here.

Also, maybe change "global" to "perServer" or some such? Not sure